### DISCUSSION TITLE
Custom tags/notes for beatmaps

### DISCUSSION THREAD

**Discussion by Phippe** [ppy/osu#13067] [Ideas]
**Describe the new feature:** Proposed by [this comment](https://blog.ppy.sh/a-definitive-lazer-faq/#comment-588): Add notes like "spaced streams" or "cat background" to a beatmap to find it faster the next time you're searching.

**Proposal designs of the feature:** Maybe add a section "Custom tags" below the already existing "Tags" section in the beatmap details, or a window similar to the one for managing collections.

**Comment by nbvdkamp** [comment]
Isn't that basically the same thing as collections?

**Comment by Phippe** [comment]
Not really, you would have to create a collection for each beatmap and then you still wouldn't be able to search for it.
